# Desired states of WBR

float64 pitch_d
float64 pitch_rate_d
float64 vel_d
float64 yaw_rate_d